pkg/admission/apiexport/admission.go

```diff
@@ -99,5 +100,25 @@ func (e *APIExportAdmission) Validate(ctx context.Context, a admission.Attribute
 		}
 	}
 
+	for i, rs := range ae.Spec.ResourceSchemas {
+		if err := validateResourceSchema(rs, field.NewPath("spec").Child("resourceSchemas").Index(i)); err != nil {
+			return admission.NewForbidden(a, err)
+		}
+	}
+
+	return nil
+}
+
+func validateResourceSchema(resourceSchema apisv1alpha2.ResourceSchema, path *field.Path) *field.Error {
+	group := resourceSchema.Group
+	if group == "" {
+		group = "core"
+	}
+
+	expectedSuffix := fmt.Sprintf(".%s.%s", resourceSchema.Name, group)
```

Comment:
this feel wrong: either call that field `resources` (then "name" makes sense), or call it `resourceSchemas` (I don't like that, but then this resource field must be called `resource` and the schema field must be `name`.

Reply:
renamed